dandi/tests/fixtures.py

```diff
@@ -52,6 +52,8 @@
     "qmri_vfa",
 ]
 
+BIDS_ERROR_TESTDATA_SELECTION = ["invalid_asl003", "invalid_pet001"]
```

Comment:
What happened to `"invalid_eeg_cbm"`?  That was one of the `dataset` values passed to `test_validate_bids_error` and `test_validate_bids_errors` prior to 6bfb52045dfcd5cef1e5ad0d681c439fce324056.

Reply:
we don't need it, it's just wasting time on more tests for the same sort of things, and as the `bids_error_examples` repository grows, it's best to already have filtering and keep the subset we use in one place rather than further down in the test function.